packages/cactus-plugin-persistence-ethereum/src/main/typescript/plugin-persistence-ethereum.ts

```diff
@@ -249,17 +248,54 @@ export class PluginPersistenceEthereum
       const checkedOwnerAddress = normalizeAddress(ownerAddress);
       const tokenUri = await tokenClient.tokenURI(tokenId);
 
+      // Fetch token metadata
+      const metadataResponse = await fetch(tokenUri);
```

Comment:
In general, we prefer using `axios` for data fetching in Cacti due to more options, could you use it instead of fetch? (low priority)